### PR TITLE
Update travis ci to force jdk8

### PR DESCRIPTION
**Pull request by ldjcmu**

### CODE REVIEW

File: catalog/build.gradle [hunk 1]
```diff
@@ -94,7 +87,9 @@ android {
     main {
       manifest.srcFile 'java/io/material/catalog/AndroidManifest.xml'
       java.srcDir 'java'
-      java.includes = srcDirs.collect { 'io/material/catalog/' + it + '/**/*.java' }
+      java.includes = srcDirs.collect {
+        'io/material/catalog/' + it + '/**/*.java'
```
Comment (pavlospt) [inline catalog/build.gradle]: Groovy supports String interpolation if you want to avoid this concatenation.

File: catalog/javatests/io/material/catalog/card/CardSwipeDismissFragmentTest.java [hunk 2]
```diff
@@ -25,7 +25,7 @@
 import static androidx.test.espresso.matcher.ViewMatchers.withText;
 
 import android.os.SystemClock;
-import androidx.fragment.app.Fragment;
+import android.support.v4.app.Fragment;
```
Comment (pavlospt) [inline catalog/javatests/io/material/catalog/card/CardSwipeDismissFragmentTest.java]: Is this change intentional?

Comment (ldjcmu) [reply catalog/javatests/io/material/catalog/card/CardSwipeDismissFragmentTest.java]: Nope O_O

**Comment by ldjcmu** [comment]
This was merged through our sync process.
